common/src/main/scala/explore/common/ObsQueries.scala

```diff
@@ -56,18 +56,24 @@ object ObsQueries {
 
   case class ScienceData(
     requirements:  ScienceRequirementsData,
-    configuration: Option[ScienceConfigurationData]
+    configuration: Option[ScienceConfigurationData],
+    constraints:   ConstraintSet
```

Comment:
Why do we need to add the constraints to `ScienceData`? I can't find where it's used.